api/categorisation.go

```diff
@@ -0,0 +1,68 @@
+package api
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-api-clients-go/v2/cantabular"
+	"github.com/ONSdigital/dp-api-clients-go/v2/cantabularmetadata"
+	"github.com/pkg/errors"
+)
+
+/*
```

Comment:
It's idiomatic to use // style comments rather than /* except when debugging/temporarily omitting code